src/@constants/index.ts

```diff
@@ -1,6 +1,20 @@
 export const FILTER_DATA = ['all', 'documentary', 'comedy', 'horror', 'crime'];
-export const SELECTION_DATA = ['sort by', 'release date'];
-export const SELECT_OPTIONS = ['Crime', 'Documentary', 'Horror', 'Comedy'];
+export const SELECTION_DATA = ['release date', 'rating', 'genre'];
+export const SELECT_OPTIONS = [
+  'Crime',
+  'Drama',
+  'Horror',
+  'Comedy',
+  'Action',
+  'Fantasy',
+  'Romance',
+  'Adventure',
+  'Documentary',
+  'Science Fiction',
+];
 
 export const API_BASE = 'http://localhost:4000/movies';
 export const API_PAGE = `${API_BASE}?offset=`;
+export const API_FILTER = `${API_BASE}?filter=`;
+export const API_SORT_BY = `${API_BASE}?sortBy=`;
```

Comment:
Imagine that you will have more of those params. Let's say 50. And imagine that at some point, you should combine those params. The common solution for such a case is to have a params builder so you can write API calls like:

```
useApiRequest(
    'delete',
   { sortBy: "ganre", offset: 2 },
    deleteMovie,
  );
```
and inside of useApiRequest:

```
const useApiRequest = (method, action, baseParams = {}, endpoint = proccess.env.API_BASE) => {
  const dispatch = useDispatch();
   
  const fetchData = useCallback(
    (params = {}, body = {}) => {
      const urlParams = buildParams({ ...baseParams, ...params });
      axios[method](`${endpoint}${urlParams}`, body)
        .then(({ data }) => {
          dispatch(action(data));
        })
        .catch((error) => {
          alert(error.message);
        });
    },
    [method, endpoint, action],
  );

  return { fetchData };
};
```

Have a look at proccess.env.API_BASE it is better to store base URL as env variable since you can have different base for deploy and dev env